pom.xml

```diff
@@ -92,7 +92,10 @@
           </execution>
         </executions>
       </plugin>
+      </plugins>
+    </pluginManagement>
 
+    <plugins>
```

Comment:
I'm not sure if this should be moved down to here (from above), but this is how it is set up in LLB.